server/graphql/v2/scalar/URL.ts

```diff
@@ -0,0 +1,17 @@
+import { GraphQLScalarType } from 'graphql';
+import { ValidationFailed } from '../../errors';
+
+export default new GraphQLScalarType({
+  name: 'URL',
+  description: "A field whose value conforms to the standard URL format as specified in RFC3986: https://www.ietf.org/rfc/rfc3986.txt.",
+  parseValue(value: string): string {
+    try {
+      return new URL(value).toString()
```

Comment:
Why is `URL` not imported? I'm curious to know how that works.

`import { URL } from 'url';`

Reply:
I noticed that in that `URL` was not imported in code sample that was [linked](https://github.com/Urigo/graphql-scalars/blob/master/src/resolvers/URL.ts) in the issue description. When i tested it out and it worked I just assumed I didn't need to add the import statement.

I will change this to include the import statement

Reply:
`URL` is in the global scope (https://nodejs.org/api/url.html). Agree that it's better to explicitly import, an ESLint rule for that could be nice.

Reply:
@Betree Ok. `URL` is effectively part of node globals. https://nodejs.org/api/globals.html

Perfect, both forms are acceptable then.